airflow/providers/databricks/sensors/databricks.py

```diff
@@ -0,0 +1,140 @@
+
+    def __init__(
+        self,
+        *,
+        databricks_conn_id: str = DatabricksSqlHook.default_conn_name,
+        http_path: Optional[str] = None,
+        sql_endpoint_name: Optional[str] = None,
+        session_configuration=None,
+        http_headers: Optional[List[Tuple[str, str]]] = None,
+        catalog: Optional[str] = None,
+        schema: Optional[str] = None,
+        client_parameters: Optional[Dict[str, Any]] = None,
+        **kwargs,
+    ) -> None:
+        """Creates a new ``DatabricksSqlSensor``."""
+        super().__init__(**kwargs)
+        self.databricks_conn_id = databricks_conn_id
+        self._http_path = http_path
+        self._sql_endpoint_name = sql_endpoint_name
+        self.session_config = session_configuration
+        self.http_headers = http_headers
+        self.catalog = catalog
+        self.schema = schema
+        self.client_parameters = client_parameters or {}
+
+    def _get_hook(self) -> DatabricksSqlHook:
+        return DatabricksSqlHook(
+            self.databricks_conn_id,
+            http_path=self._http_path,
+            session_configuration=self.session_config,
+            sql_endpoint_name=self._sql_endpoint_name,
+            http_headers=self.http_headers,
+            catalog=self.catalog,
+            schema=self.schema,
+            **self.client_parameters,
+        )
+
+
+class DatabricksPartitionTableSensor(DatabricksBaseSensor):
+    """
+    Waits for a partition to show up in Databricks.
+
+    :param table: The name of the table to wait for, supports the dot
+        notation (my_database.my_table)
+    :param partition: The partition clause to wait for.
+    :param database: The name of the database in Databricks. It uses 'default' if nothing is provided
+    :param databricks_conn_id: Reference to the :ref:`Databricks connection <howto/connection:databricks>`.
+    """
+    template_fields: Sequence[str] = (
+        'database',
+        'table',
+        'partition',
+    )
+
+    def __init__(self, *,
+                 databricks_conn_id: str,
+                 table: str,
+                 partition: str,
+                 database: Optional[str] = 'default',
+                 **kwargs: Any):
+        super().__init__(**kwargs)
+        self.databricks_conn_id = databricks_conn_id
+        self.table = table
+        self.partition = partition
+        self.database = database
+
+    def poke(self, context: Context) -> bool:
+        hook = self._get_hook()
+        _, result = hook.run(f'SHOW PARTITIONS {self.database}.{self.table}')
+        record = result[0] if result else {}
+        return self.partition in record
+
+
+class DatabricksDeltaTableChangeSensor(DatabricksBaseSensor):
+    """
+    Waits for Delta table event
+
+    :param table: The name of the table to wait for, supports the dot
+        notation (my_database.my_table)
+    :param timestamp: The timestamp that will be used to filter new events.
```

Comment:
Nitpick:  Maybe include the expected format of the timestamp?

Reply:
Fixed.